src/main/java/mops/services/webServices/WebApplicantService.java

```diff
@@ -0,0 +1,7 @@
+package mops.services.webServices;
```

Comment:
Soll hier was rein? wenn nicht bitte löschen

Reply:
also in die Klasse

Reply:
removed